config/migrations/20230405150000-add-agenda-status-activities-for-kanselarij-graph.sparql

```diff
@@ -0,0 +1,29 @@
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX besluitvorming: <http://data.vlaanderen.be/ns/besluitvorming#>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX generiek: <http://data.vlaanderen.be/ns/generiek#>
+
+CONSTRUCT {
+  GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
+     ?newAgendaStatusActivity a ext:AgendaStatusActivity ;
+                                              mu:uuid ?uuid ;
+                                              prov:used ?agenda ;
+                                              prov:startedAtTime ?modified ;
+                                              generiek:bewerking ?agendaStatus .
+  }
```

Comment:
Deze `CONSTRUCT` gaat niet werken. Ik zie dat het zo in de ticket beschreven staat, maar ik vermoed dat het een foutje/oversight is. Je moet een `INSERT` statement gebruiken, zoals in de andere migratie.

Voor een klein beetje context: `CONSTRUCT` queries gaan (een beetje confusingly) geen nieuwe data aanmaken, maar ze laten je wel toe om data in graaf-vorm terug te krijgen. Waar een `SELECT` query een tabel met rows (de resultaten) en kolommen (de variabelen die je selecteert) returnt, gaat een `CONSTRUCT` query een graaf in bv. RDF returnen.

Reply:
Kan je de nieuwe activiteiten ook ineens `prov:Activity` meegeven als type? Dit gaat vanzelf gebeuren voor nieuwe activiteiten die vanuit de frontend aangemaakt worden door mu-cl-resources, maar deze gaan ze niet hebben tenzij je die expliciet toevoegt in de migratie.

Dit kan je doen door bv. `?newAgendaStatusActicity a prov:Activity, ext:AgendaStatusActivity` te schrijven.